### ISSUE TITLE
A11y_SQLAzureDataStudio_NewServerGroup_AddServerGroup_ScreenReader:The name of the dialog is not announced as visually seen.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9228]
[“Check out Accessibility Insights](https://accessibilityinsights.io/)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOs;#MAS1.3.1;#VoiceOver; 

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Mac OS:10.15.3
Screen reader: Voiceover
MAS References:MAS1.3.1

**Repro Steps**:
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Launch screen reader as voiceover.
4. Hover over the 'Servers' header and select the 'New Server Group' icon.

**Actual:**
When the dialog opens screen reader announces it as "Server Group" dialog

**Expected:**
Screen reader should read the name as "Add Server group" dialog as seen on screen.

**User Impact:**
Screen reader users will not receive the right name of the dialog, they will not know that the dialog is for adding server groups.

Attachment for Reference :
![image](https://user-images.githubusercontent.com/24580570/74927543-d1ce2200-53fd-11ea-97c4-00f24e7aa556.png)

**Comment by alanrenmsft** [comment]
the dialog aria-label should match the displayed title

**Comment by BhanuPrakashGit** [comment]
Verified the bug, issue is not repro'ing.
GithubTags:#Closed;
PFA for reference
[9228_Closed.zip](https://github.com/microsoft/azuredatastudio/files/10742109/9228_Closed.zip)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
